newsletter-january-2023.rst

```diff
@@ -0,0 +1,83 @@
+.. post:: Jan 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - January 2023
+=======================================
+
+Happy 2023! In this newsletter, we present the updates happening in 2022's last month and since the :doc:`previous newsletter </newsletter-december-2022>`.
+
+.. Commenting out this stuff, isn't it better that 
+.. 
+.. News and updates
+.. ----------------
+.. 
+.. Here are the latest updates from our team:
```

Comment:
I think this section gives some context to the following bullets. Otherwise, I'm not sure it feels fine to start reading a bunch of items without too much context about what they are.

Reply:
Aren't you commenting on something that I commented _out_ (because of the same reasons you also stated) ? :)

Reply:
It seems you commented it to remove it. My point is that we should keep it because it works fine.

Reply:
Sure, I can bring it back. I wanted to remove it to get the list further up on the screen.

Writing "Newsletter", then an intro, then "News and updates", then "Here are the latest updates from our team" just seemed like a slow start. Thought it'd be more simple to present the list as quickly as possible.

Reply:
I re-added it so we can get the newsletter out. I still don't like to have such a long intro, but I'll repeat the proposal in the next newsletter.